server/app/controllers/api/v1/api_users_controller.rb

```diff
@@ -1,10 +1,11 @@
 class Api::V1::ApiUsersController < ApplicationController
   before_action :set_api_user, only: [:show, :update, :destroy]
-  #xbefore_action :set_default_response_format
+
   
   #GET /api_users
   def index
-    @api_users = ApiUser.includes(:wishes).all
+    @api_users = ApiUser.includes(:wishes).includes(wishes: :programminglanguage).includes(wishes: :meetinginterval).all
```

Comment:
Perhaps the ApiUser should be sorted by username or something else. Currently they will be in whatever order the database decides. 

This isn't really a problem with your change, as this was also a "problem" before.

Reply:
index created fro api users, based on email and mentor/mentee

Reply:
index will not change the order. It might improve speed and ensure consistency (depending on how you would create it), but order will still be whatever order database returns the data in.